src/control/cmd/dmg/pool.go

```diff
@@ -56,12 +56,12 @@ type PoolCreateCmd struct {
 	Properties    PoolSetPropsFlag `short:"P" long:"properties" description:"Pool properties to be set"`
 	ACLFile       string           `short:"a" long:"acl-file" description:"Access Control List file path for DAOS pool"`
 	Size          string           `short:"z" long:"size" description:"Total size of DAOS pool or its percentage ratio (auto)"`
-	TierRatio     string           `short:"t" long:"tier-ratio" default:"6,94" description:"Percentage of storage tiers for pool storage (auto)"`
+	TierRatio     string           `short:"t" long:"tier-ratio" default:"6% SCM, 94% NVMe" description:"Percentage of storage tiers for pool storage (auto)"`
```

Comment:
this seems to break the tier ratio parsing at https://github.com/daos-stack/daos/blob/cde0a4caf397eb731b8f64fd7f20e2bc2a6d51b7/src/control/cmd/dmg/pool.go#L180 ... should we go back to the original "6,94" ?

Reply:
Ah, hmm. Yes, we should probably improve that parsing. Allowing floats would be a bigger change, but at the least we could filter out the '%' character before parsing.

Reply:
i just reverted that today and went back to the original "6,94" integer array...